### DISCUSSION TITLE
[match] AWS Secrets Manager for storage

### DISCUSSION THREAD

**Discussion by mriccia** [fastlane/fastlane#18550] [Ideas]
Currently signing identities can be stored in a Git repo, Google Cloud or Amazon S3.

It would be great to be able to store the identities in AWS Secrets Manager, which has specific additional features for Secrets management.

**Comment by ctaintor** [comment]
hey @mriccia – we wrote https://github.com/klarna-incubator/fastlane-plugin-secrets_manager_storage to do exactly this. If it's still useful to you, check it out and let me know if you hit any problems. Also happy to take feedback on the README :)
